### PR TITLE
MAT-4695 & MAT-4693 & some minor regression timing fixes

### PR DESCRIPTION
**Pull request by bwelch8105**
Made some minor adjustments to the following files to resolve some minor timing issues:
-- TestCasePopulationValues.spec.ts
-- StratificationExpectedValues.spec.ts

Additionally, I added a new file:
-- PassingCodeCoverageTestCaseListPage.spec.ts
to cover a test case that can be automated,
in relation to user stories MAT-4695 and MAT-4693. I also updated the following file to support the new the new test spec file for those user stories:
--TestCasesPage.ts

### CODE REVIEW

File: cypress/integration/WebInterface/Test Cases/TestCasePopulationValues.spec.ts [hunk 1]
```diff
@@ -604,7 +603,7 @@ describe('TC Pop value options are limited to those that are defined from Measur
         cy.get(EditMeasurePage.cqlEditorTab).click()
         cy.get(EditMeasurePage.cqlEditorTextBox).type('{enter}')
         cy.get(EditMeasurePage.cqlEditorSaveButton).click()
-        cy.wait(4500)
+        cy.wait(14500)
```
Comment (cvasile001) [inline cypress/integration/WebInterface/Test Cases/TestCasePopulationValues.spec.ts:606]: is it possible to wait for an element vs an explicit wait?

File: cypress/integration/WebInterface/Test Cases/Validations/PassingCodeCoverageTestCaseListPage.spec.ts [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+        MeasureGroupPage.CreateProportionMeasureGroupAPI(false, false, 'Initial Population', 'Initial Population', 'Initial Population', 'Boolean')
+        OktaLogin.Login()
+    })
+    afterEach('Logout', () => {
+        OktaLogin.Logout()
+        Utilities.deleteMeasure(measureName, CqlLibraryName)
+
+    })
+
+    it('Validate Passing and Code Coverage tabs contain the initial "-" value, and displayes a percentage when Test Case is ran', () => {
```
Comment (cvasile001) [inline cypress/integration/WebInterface/Test Cases/Validations/PassingCodeCoverageTestCaseListPage.spec.ts:50]: are we validating a specific expected percentage?

File: cypress/integration/WebInterface/Test Cases/Validations/StratificationExpectedValues.spec.ts [hunk 3]
```diff
@@ -523,6 +523,11 @@ describe('Stratification Expected values for Non Boolean Population Basis', () =
         cy.get(TestCasesPage.runTestButton).should('be.enabled')
         cy.get(TestCasesPage.runTestButton).click()
 
+        //validate color of Measure Group-1 Stratification after running
```
Comment (cvasile001) [inline cypress/integration/WebInterface/Test Cases/Validations/StratificationExpectedValues.spec.ts:526]: I would just drop this check, its a bit too granular for us for now. And we know it is going to be changing soon

**Review by cvasile001 — CHANGES_REQUESTED** [review]
left a few comments/questions

File: cypress/integration/WebInterface/Test Cases/Validations/PassingCodeCoverageTestCaseListPage.spec.ts (hunk 2 above)
Comment (bwelch8105) [reply cypress/integration/WebInterface/Test Cases/Validations/PassingCodeCoverageTestCaseListPage.spec.ts:50]: We aren't waiting for specific percentages, yet, that part of the percentages stuff is supposed to come in a later story, is what Prateek was telling me. At this time, we just needed to validate that, initially, it shows that dash ("-") value and then some randomly generated value after the "Execute Test Case" button is clicked. I manually validated the changing values of what appears after the button is clicked and I just made the automation verify that the dash ("-") no longer is present after the "Execute Test Case" button is clicked.

File: cypress/integration/WebInterface/Test Cases/Validations/StratificationExpectedValues.spec.ts (hunk 3 above)
Comment (bwelch8105) [reply cypress/integration/WebInterface/Test Cases/Validations/StratificationExpectedValues.spec.ts:526]: Ok.
I will remove this

File: cypress/integration/WebInterface/Test Cases/TestCasePopulationValues.spec.ts (hunk 1 above)
Comment (bwelch8105) [reply cypress/integration/WebInterface/Test Cases/TestCasePopulationValues.spec.ts:606]: I don't like having to put that wait in either but it is for a wonlky issue around first creating the measure with CQL, on the back end, and, then, trying to add or adjust a group through the UI. We are having to put that whole block where we gotta log in, put some random change in the CQL, save that change and, then, wait for the change to take affect. I am seeing that it is taking longer and longer for that occur, hence the increase on the explicit wait. (edited) 
[2:09](https://semanticbits.slack.com/archives/G017E78C1FY/p1666033781665599)
If we don't do that whole step of logging in, after creating the measure with CQL and all via API call, and, then, immediately dive into adjusting a group or adding one to the measure, then we get invalid errors around the scoring not matching up with what is in the CQL.

**Pull request merged**